src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -14,7 +14,7 @@ import { CommandListArgs } from '../list/list.js';
 
 const SHEET_MIN_X = MapSheet.origin.x + 4 * MapSheet.width; // The minimum x coordinate of a valid sheet / tile
```

Comment:
I am not sure why we even have these, there is a MapSheet object with more precise  min and max values, which also have included unit tests.

We should remove what ever is referencing these.

Reply:
We could consult with the Land team and see what their requirements are? Perhaps we need to step back and look at the whole of tileset validation as we have another ticket regarding logging https://toitutewhenua.atlassian.net/browse/TDE-1013